app/javascript/packs/application/components/PointsList.js

```diff
@@ -20,7 +21,7 @@ export class PointsList extends Component {
     }
 
     return (
-      <div>{points.map(point => <h4 key={point.id}>{point.caption}</h4>)}</div>
+      <div>{points.map(point => <Point point={point} key={point.id} />)}</div>
```

Comment:
When using ids for keys, I usually like to put a string on the front to decrease the possibility for React throwing a warning about needing unique keys (this [page](https://reactjs.org/docs/lists-and-keys.html#basic-list-component) talks about that warning).  Probably not necessary if you haven't seen the warning, but something that I think about when setting `key` props in React.